content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,39 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+
+**Features:**
+
+- Re-introducing the experimental [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio). Cypress Studio provides a visual way to generate tests within Cypress by recording interactions against the application under test. Cypress Studio must be enabled via the `experimentalStudio` flag and is currently not supported in component tests. Addresses [#23461](https://github.com/cypress-io/cypress/issues/23461)
```

Comment:
I think changelogs tend to be written past tense... eg "re-introduced" etc (just based on our other change logs, not sure if there's a hard/fast rule)

Reply:
At least a more passive present tense.

```suggestion
- Re-introduces the experimental [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio). Cypress Studio provides a visual way to generate tests within Cypress by recording interactions against the application under test. Cypress Studio is available only in end-to-end tests, and must be enabled via the `experimentalStudio` flag. Addresses [#23461](https://github.com/cypress-io/cypress/issues/23461)
```